### PR TITLE
Settings page redesign! (+ popups update)

### PR DESCRIPTION
**Pull request by GDUcrash**
* Light mode! The default one is still dark but it can be changed in settings. Works with popups too
* Changed the icons to be from one icon set (unicons)
* Minor css tweaks
* Please excuse my weird js

### CODE REVIEW

File: popups/cloud-games/popup.js [hunk 1]
```diff
@@ -1,3 +1,16 @@
+//theme
+const lightThemeLink = document.createElement("link");
+lightThemeLink.setAttribute("rel", "stylesheet");
+lightThemeLink.setAttribute("href", "light.css");
+
+chrome.storage.sync.get(["globalTheme"], function (r) {
+  let rr = false; //true = light, false = dark
+  if (r.globalTheme) rr = r.globalTheme;
+  if (rr) {
```
Comment (WorldLanguages) [inline popups/cloud-games/popup.js:9]: This JS is weird, but it works I guess.

File: popups/scratch-messaging/light.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  background: white;
+  color: #222;
+}
+
+button {
+  border: 1px solid #8f8f8f;
+  background: #e0e0e0;
+  color: #222;
+}
+body::-webkit-scrollbar-track {
```
Comment (WorldLanguages) [inline popups/scratch-messaging/light.css:11]: Did you style Mozilla's scrollbar?

**Comment by WorldLanguages** [comment]
The constrast on here is barely noticeable, unlike dark mode
image

**Comment by WorldLanguages** [comment]
Popout icon stays white on light mode:
image

**Comment by WorldLanguages** [comment]
Let's use a single uppercase pattern on here? 
image

**Comment by WorldLanguages** [comment]
4. Should we use this opportunity to implement #498? I would also like to add some non-addon-specific settings when adding the ability to mute notifications.

**Comment by WorldLanguages** [comment]
5. (not 100% serious) I feel no one will use light mode, it will become a meme like Discord's, and only @JeffaloBob will use it :P

**Comment by YOYITsM3M8** [comment]
haha

**Comment by Hans5958** [comment]
Why not #FFF white?

**Comment by Hans5958** [comment]
> 
> 
>     1. (not 100% serious) I feel no one will use light mode, it will become a meme like Discord's, and only @JeffaloBob will use it :P

I beg to differ. To be honest this should be the default theme just like MSU.

**Comment by WorldLanguages** [comment]
@Hans5958 I see your point, but it's 2020! Dark mode is now the default. If Scratch had the resources for a dark mode I'm sure they would offer one.

**Comment by Hans5958** [comment]
> It's 2020! Dark mode is now the default.
Bold claim, but you sure about that? Why we still have the light theme as the default anywhere, then?

**Comment by Hans5958** [comment]
> Why not #FFF white?

@GDUcrash

**Comment by GDUcrash** [comment]
some parts are #fff

**Comment by Hans5958** [comment]
Wouldn't it better to have the background to white, and make the elements either darker (like #eee) or have borders?

**Comment by mxmou** [comment]
> Wouldn't it better to have the background to white, and make the elements either darker (like #eee) or have borders? I don't know.

Black (or almost black) text is easier to read on #eee than on #fff, at least for some people.

**Comment by Explosion-Scratch** [comment]
Thanks for doing this Ucrash! It looks amazing!

**Pull request merged**